dev-data/widget-communication.data.ts

```diff
@@ -0,0 +1,31 @@
+export const communicationText = {
```

Comment:
We need to come up with some naming rules for cases like this https://github.com/rolling-scopes/site/pull/538#discussion_r1773365305

Reply:
Sure. Don't know how to name this better, but as long, as our next step is to refactor dev-data, I suppose any "ok" name fits.